client/components/Badges.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {Icon, Segment, List, Grid} from 'semantic-ui-react';
+import axios from 'axios';
+import { VerifiedUserRounded, StarRounded, CheckCircleRounded } from '@material-ui/icons';
+// import VerifiedUserRoundedIcon from '@material-ui/icons/Veri';
+
+const iconStyle = {
+  color: '#FF385C',
+  verticalAlign: 'bottom'
+};
+
+const contentStyle = {
+  marginLeft: '8px',
+  display: 'inline'
+}
+
+class Badges extends React.Component {
+
+  constructor(props) {
+  super(props);
+  this.state = {
+    reviews: null
+    }
+  }
+
+  componentDidMount(){
+
+    const url = 'http://ec2-3-129-14-177.us-east-2.compute.amazonaws.com:8080/api/reviews';
+    // const url = 'http://localhost:2000/api/reviews';
+    const options = {
+      params: { array: JSON.stringify(this.props.listingIds)},
+  
+    };
+
```

Comment:
Hey did you manage to get this endpoint to work?

Reply:
It was working so depends on the error. Most likely it might be the ec2 instance itself like yesterday,